pyproject.toml

```
@@ -62,29 +62,29 @@ redis = [
     "redis[hiredis]~=5.0.1",
 ]
 mongo = [
-    "motor~=3.3.2",
+    "motor~=3.5.1",
```

Does version 3.3 failed tests with the current version?

```suggestion
    "motor>=3.3,<3.6",
```